src/components/Search/index.tsx

```diff
@@ -265,8 +265,6 @@ function Search({
     const {accountID, email} = useCurrentUserPersonalDetails();
     const [isActionLoadingSet = new Set<string>()] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_METADATA}`, {canBeMissing: true, selector: isActionLoadingSetSelector});
 
-    const {markReportIDAsMultiTransactionExpense, unmarkReportIDAsMultiTransactionExpense} = useContext(WideRHPContext);
```

Comment:
For interest of the reviewer, this is where the small conflict was - there was a moved declaration of one variable next to this line we are removing so it was ok to remove it, the same conflict on the original PR is here https://github.com/Expensify/App/pull/75713#discussion_r2550590008